base_report_to_printer/printing.py

```diff
@@ -127,10 +128,23 @@ def print_document(self, report, content, format):
             os.write(fd, content)
         finally:
             os.close(fd)
-        connection = cups.Connection()
+
+        try:
+            _logger.debug(
+                'Starting to connect to CUPS on %s:%s'
+                % (CUPS_HOST, CUPS_PORT))
+            connection = cups.Connection(CUPS_HOST, CUPS_PORT)
+            _logger.debug('Connection to CUPS successfull')
+        except:
+            raise Warning(
+                _("Failed to connect to the CUPS server on %s:%s. "
+                    "Check that the CUPS server is running and that "
```

Comment:
@alexis-via it's just about the string, when we break them into several lines  we usually align all lines to the same indent as the first `"`.